gradio_tts_app.py

```
@@ -272,103 +299,134 @@ def load_and_prepare_model():
-        logger.info(f"   Using reference audio: {audio_prompt_path}")
-    else:
-        logger.info("   Using default voice")
-    
-    logger.info(f"   Parameters: temp={config.temperature}, cfg={config.cfg_weight}, exaggeration={config.exaggeration}, steps={config.steps}")
-    start_time = time.time()
+def _combine_audio_chunks(audio_chunks: List[np.ndarray]) -> np.ndarray:
+    """Combine multiple audio chunks into a single array."""
+    if len(audio_chunks) > 1:
+        return np.concatenate(audio_chunks)
+    elif audio_chunks:
+        return audio_chunks[0]
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)